0001/README.md

```diff
@@ -253,18 +254,18 @@ We separate *client authentication* and *endpoint identification*.
 During client authentication, client and server should verify each other: client verifies server identity and server verifies that client is authorized to access server.
 Note that multiple clients MAY share the same credentials, in which case server can not identify clients: it merely verifies that client knows a shared secret.
 
-Client authentication and *session-wide encryption* are usually handled simultaneously by separate encapsulating protocols (TLS, DTLS, IPSec, none, etc.), so it is not described in this RFC.
-However, a possible authentication method is using MQTT CONNECT packet with name and password.
-It is also possible to disable client authentication at all, but that's insecure and is not recommended.
+Client authentication and *session-wide encryption* are usually handled simultaneously by separate encapsulating protocols (TLS, DTLS, IPSec, none, etc.), and are therefore not described in this RFC.
+However, a possible authentication method would use an MQTT CONNECT packet with name and password.
+You can also completely disable client authentication, but that's insecure and is not recommended.
 
-After the client authentication is done, client may issue commands on behalf of endpoints.
-To do so, client needs an [endpoint token](#language).
+Upon successful client authentication, the client may issue commands on behalf of the endpoints.
```

Comment:
Why "the"? Are we talking about particular clients/endpoints here?

Reply:
Yes, about the successfully authenticated ones.

Reply:
Gotcha.